docs/guides/private-python-packages.rst

```diff
@@ -0,0 +1,112 @@
+Installing Private Python Packages
+==================================
+
+.. warning::
+
+   This guide is for :doc:`/commercial/index`.
+
+Read the Docs uses :doc:`pip <pip:index>` to install your Python packages.
+If you have private dependencies, you can install them from
+a :ref:`private Git repository <guides/private-python-packages:From a Git repository>` or
+a :ref:`private repository manager <guides/private-python-packages:From a repository manager other than PyPI>`.
+
+From a Git repository
+---------------------
+
+Pip supports installing packages from a :ref:`Git repository <pip:vcs support>` using the URI form:
+
+.. code::
+
+   git+https://gitprovider.com/user/project.git@{version}#egg={package-name}
```

Comment:
I think an example would be useful:

```suggestion
   git+https://gitprovider.com/user/project.git@{version}#egg={package-name}
   # For example
   # git+https://github.com/readthedocs/sphinx-hoverxref@master#egg=hoverxref
```

I don't think `#egg=` is always needed -- I think it's mostly confusing to users, so might be worth leaving off?